src/Presentation/ExportConfigEditPageTextBuilder.php

```diff
@@ -17,144 +19,210 @@ public function __construct(
-				'wikibase-export-config-help-variable-subject-filter-property-id',
-				'"P50"',
-			) .
-			$this->createTableRow(
-				'subjectFilterPropertyValue',
-				'wikibase-export-config-help-variable-subject-filter-property-value',
-				'"company"',
-			) .
-			$this->createTableRow(
-				'exportLanguages',
-				'wikibase-export-config-help-variable-export-languages',
-				'[ "en", "nl" ]',
-			) .
-			'</tbody>' .
-			'</table>';
+	public function createBottomHtml(): string {
+		return <<<HTML
+<div id="Documentation">
+	<section>
+		<h2 id="ConfigurationDocumentation">{$this->context->msg( 'wikibase-export-config-help' )->escaped()}</h2>
+
+		<p>
+			Besides the configuration reference below, you can consult the Wikibase Export
+			<a href="https://professional.wiki/en/extension/wikibase-export">usage documentation</a> and
+			<a href="https://export.wikibase.wiki">demo wiki</a>.
+		</p>
+	</section>
+
+	<section>
+		<h2 id="ExportLanguage">Export language</h2>
+
+		<p>
+			By default the export happens in the wikis main language.
+		</p>
+
+		<p>
+			You can change the export language or let the user choose by specifying multiple available languages.
+		</p>
+
+		<p>
+			Example configuration:
+		</p>
+
+		<pre>
+{
+	"exportLanguages": [
+		"en",
+		"nl",
+		"de"
+	]
+}</pre>
+	</section>
+
+	<section>
+		<h2 id="ExportSubjects">Export subjects</h2>
+
+		<p>
+			You can specify the subjects that should be selected for export by default.
+		</p>
+
+		<pre>
+{
+    "defaultSubjects": [
+        "Q1",
+        "Q2"
+    ]
+}</pre>
+
+		<p>
+			This can be any entities defined in this wiki.
+		</p>
+
+		<p>
+			You can also limit which entities show up in the search results via
+			<code>subjectFilterPropertyId</code> and <code>subjectFilterPropertyValue</code>. Example:
+		</p>
+
+		<pre>
+{
+    "subjectFilterPropertyId": "P1",
+    "subjectFilterPropertyValue": "Q2"
+}</pre>
+	</section>
+
+	<section>
+		<h2 id="UngroupedValues">Exporting ungrouped values</h2>
+
+		<p>
+			To have the "ungrouped values" section show on the export page, configure at least one ungroued property.
+		</p>
+
+		<pre>
+{
+    "ungroupedProperties": [
+        "P1",
+        "P2"
+    ]
+}</pre>
+
+		<p>
+			Statements with ungrouped properties will have their values be included in the export without
+			grouping or filtering.
+		</p>
+	</section>
+
+	<section>
+		<h2 id="GroupingByYear">Grouping values by year</h2>
+
+		<p>
+			Values can be filtered and grouped by year. You can configure which properties should have their
+			values be grouped this way.
+		</p>
+
+		<pre>
+{
+    "propertiesToGroupByYear": [
+        "P3",
+        "P4"
+    ]
+}</pre>
+
+		<p>
+			For the "values grouped by year" section to show on the export page, you need to configure at least
+			one property to be grouped by year and at least one time qualifier property:
+		</p>
+
+		<pre>
+{
+    "startTimePropertyId": "P10",
+    "endTimePropertyId": "P11"
+}</pre>
```

Comment:
Should we mention `pointInTimePropertyId` in this section? I know that is an example, but all the other variables are at least mentioned somewhere.

Reply:
Yes, we should. Nice catch